<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pSimulationParam">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Simulation</string>
     </attribute>
     <widget class="QLabel" name="lblTemperature">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsbTemperature">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="decimals">
       <number>10</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10000000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbUpdatesNumber">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblUpdatesNumber">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Updates number:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsbSynchronyRate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="lblSynchronyRate">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Synchrony rate:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbSimulationType">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Simulation type:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbStart">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>80</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbStop">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>80</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>St&amp;op</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkUniformalTemperature">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>80</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>Uniformal Temperature</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblSpeedPercent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>SpeedPercent</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbSpeedPercent">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pNeuronsParam">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Neurons parameters</string>
     </attribute>
     <widget class="QLabel" name="lblThreshold">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsbThreshold">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>10</number>
      </property>
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.001000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="lblState">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>57</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>State:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblNeuronTemperature">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsbNeuronTemperature">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>10</number>
      </property>
      <property name="maximum">
       <double>10000000000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbNbStates">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblStateNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of states:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbState">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNodeID">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>61</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Node Id:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrent">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Index:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtIndex">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pSynapsesParam">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Synapses parameters</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="dsbWeight">
      <property name="geometry">
       <rect>
        <x>111</x>
        <y>10</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-99.989999999999995</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="lblWeight">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Weight:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbDelay">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblDelay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>48</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Delay:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pUpdateSchedule">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>293</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Update schedule</string>
     </attribute>
     <widget class="QLabel" name="lblUpdateBlock">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Update block:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblUpdateMethods">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Methods:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblScheduleType">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Type of schedule:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbUpdateBlock">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbScheduleType">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>91</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbUpdateMethods">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="DesignPlan" name="frmDesign" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>551</width>
      <height>461</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="actionO_pen"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExport_To_GNBox_Premodel"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionO_pen">
   <property name="text">
    <string>O&amp;pen</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionSelect_borders">
   <property name="text">
    <string>Select borders</string>
   </property>
  </action>
  <action name="actionSelect_centers">
   <property name="text">
    <string>Select centers</string>
   </property>
  </action>
  <action name="actionKernel">
   <property name="text">
    <string>Kernel</string>
   </property>
  </action>
  <action name="actionExport_To_GNBox_Premodel">
   <property name="text">
    <string>Export To &amp;GNBox Premodel...</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DesignPlan</class>
   <extends>QWidget</extends>
   <header>designplan.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
